using_images/s2i_images/dot_net_core.adoc

```diff
@@ -33,22 +33,27 @@ and Python from within {product-title}.
 [[dot-net-core-supported-versions]]
 == Supported Versions
 
-* .NET Core version 1.0
+* .NET Core version 2.0
 * .NET Core version 1.1
+* .NET Core version 1.0
 * Supported on Red Hat Enterprise Linux (RHEL) 7
 ifdef::openshift-enterprise[]
 and {product-title} versions 3.3 and later
 endif::openshift-enterprise[]
 
-The .NET Core software collection (`rh-dotnetcore10` and `rh-dotnetcore11`)
-ships with the *_project.json_* build system (`1.0.0-preview2` SDK). See the
-Known Issues chapter in the .NET Core
-link:https://access.redhat.com/documentation/en/net-core/1.1/paged/release-notes/[Release
-Notes] for details on installing this SDK on a non-RHEL system.
+The .NET Core software collection for version 2.0 supports the
+*_msbuild/csproj_* build system. Versions 1.1 and 1.0 (`rh-dotnetcore11` and
+`rh-dotnetcore10`) ships with the obsolete *_project.json_* build system
```

Comment:
I don't think we should use "obsolete" for 1.1 and 1.0. The project.json files are correct for those versions.

Reply:
We can leave the word out, or use deprecated instead?